terraform-build-modules/index.json

```diff
@@ -41,5 +42,5 @@
   "backend": {
     "imageid": "ubuntu"
```

Comment:
The vanilla ubuntu image returns a Python 3.5 deprecation warning, but testing with `ubuntu:2004` did not. More to do with the katacoda image than the content though.

<img width="965" alt="Screen Shot 2021-02-24 at 7 28 12 PM" src="https://user-images.githubusercontent.com/8883519/109085222-430e7800-76d7-11eb-96ef-e1c74c44acc5.png">